dashboard/app/main.go

```diff
@@ -1436,6 +1455,10 @@ func handleSubsystemsList(c context.Context, w http.ResponseWriter, r *http.Requ
 	if service == nil {
 		return fmt.Errorf("%w: the namespace does not have subsystems", ErrClientBadRequest)
 	}
+	if r.FormValue("json") == "1" {
+		w.Header().Set("Content-Type", "application/json")
+		return ssListJSON(service, w)
```

Comment:
Why not do it via `public_json_api.go` like it's done for other `json=1` cases?

Reply:
Because I don't want others to depend on this code.
We did it before in multiple places -> [here](https://github.com/search?q=repo%3Agoogle%2Fsyzkaller%20formvalue(%22json%22)&type=code).

Reply:
We can additionally control the referrer... but it is probably too much...